cluster/vagrant/gluster-daemonset.yaml

```diff
@@ -0,0 +1,105 @@
+---
+kind: DaemonSet
```

Comment:
Why is the gluster server not made accessible via a service and only runs once in the cluster? This is not an HA gluster setup, right?